### PR TITLE
#165305709 Set up empty react project

### PR DESCRIPTION
**Pull request by Philipotieno**
#### What does this PR do?
 -  Set up empty react project
#### Description of Task to be completed?
 - Create a react project from scratch using Webpack 4, Babel 7.
#### How should this be manually tested?
- `$ git clone https://github.com/andela/ah-the-immortals-frontend`
- `$ git fetch origin` - To get all branches
- `$ git checkout ch-set-up-react-165305709`
- `$ npm install`
- `$  npm run lint src
- `$ npm run start-server`
#### Screenshots
image

PT
[#165305709](https://www.pivotaltracker.com/n/projects/2326460/stories/165305709)

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1 @@
+<script type="text/javascript" src="index_bundle.js"></script>
```
Comment (IssaIan) [inline dist/index.html]: @Philipotieno, the dist folder should be added to gitignore. It should not be pushed.

**Review by IssaIan — CHANGES_REQUESTED** [review]

File: dist/index.html (hunk 1 above)
Comment (Philipotieno) [reply dist/index.html]: I am fixing that now @IssaIan

**Review by Ogutu-Brian — APPROVED** [review]

**Review by Bakley — APPROVED** [review]
@Philipotieno Good job on the setup. Cheers

**Review by IssaIan — APPROVED** [review]
LGTM

**Comment by koechkevin** [comment]
LGTM

**Pull request merged**